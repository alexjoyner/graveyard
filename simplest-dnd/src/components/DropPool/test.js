import React from 'react';
import {shallow} from 'enzyme';
import {DropPool} from '.';


describe('Drop Pool', () => {
    let component;
    beforeEach(() => {
        const props = {
            connectDropTarget: el => el
        };
        component = shallow(<DropPool {...props}/>);
    });
    it('should render properly', () => {
        expect(component).toMatchSnapshot();
    });

});