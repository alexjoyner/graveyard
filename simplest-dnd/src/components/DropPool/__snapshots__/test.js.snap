// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drop Pool should render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DropPool
    connectDropTarget={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "Drop Pool",
        Array [
          <h1>
            Box
          </h1>,
        ],
      ],
      "className": "drop-pool",
    },
    "ref": null,
    "rendered": Array [
      "Drop Pool",
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "host",
        "props": Object {
          "children": "Box",
        },
        "ref": null,
        "rendered": "Box",
        "type": "h1",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "Drop Pool",
          Array [
            <h1>
              Box
            </h1>,
          ],
        ],
        "className": "drop-pool",
      },
      "ref": null,
      "rendered": Array [
        "Drop Pool",
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "host",
          "props": Object {
            "children": "Box",
          },
          "ref": null,
          "rendered": "Box",
          "type": "h1",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
